### DISCUSSION TITLE
Support WFRP4 for Module Item Piles

### DISCUSSION THREAD

**Discussion by GregorHak** [moo-man/WFRP4e-FoundryVTT#1052] [General]
Hey, my group and I want to use Item Piles module, but we do not know how to configure it correctly. There is option to Requesting New Systems, but we must complete our application. We have no knowledge to fill in the form, can you help? 
[Link to Item Piles module](https://foundryvtt.com/packages/item-piles). Foundry page link to wfrp4 system is easy. Actor Class Type we suppose that is "character" and Item Quantity Attribute is "data.quantity", but we not sure. Item filters need attribute path, we need it for Career, Critical, Disease, injury, Mutation, Prayer, Psychology, Talent, Skill, Spell, Trait, Extended Test, Vehicle Modification, Cargo. Similarities and Currencies we think that can stay blank, because wfrp4 do not need this. Somebody can help? Maybe Moo Man can help?

# **[INSERT FOUNDRY PAGE LINK TO SYSTEM HERE]**

### **Actor Class Type**: "[WRITE TYPE HERE]"

*The actor class type is the type of actor that will be used for the default item pile actor that is created on first item drop.*

D&D5e system example: `character`

### **Item Quantity Attribute**: "[WRITE ATTRIBUTE PATH HERE]"

*The item quantity attribute is the path to the attribute on items that denote how many of that item that exists.*

D&D5e system example: `data.quantity`

### **Item Filters**: "[WRITE ITEM FILTERS HERE]"

*Item filters actively remove items from the item pile inventory UI that users cannot loot, such as spells, feats, and classes.*

D&D5e system example: `spell, feat, class`

### **Item Similarities**: [WRITE ITEM SIMILARITIES HERE]

*Item similarities determines how item piles detect similarities and differences in the system.*

D&D5e system example: `name, type`

### **Currencies**:

*Currencies in item piles are a list of names, attribute paths, and images - the attribute path is relative to the actor.data.*

- **Name**: [WRITE ATTRIBUTE NAME HERE]
    - **Path**: [WRITE ATTRIBUTE PATH HERE]
    - **Image**: [WRITE IMAGE PATH HERE (or blank)]
    
D&D5e system example:

- **Name**: "Platinum Coins",
    - **Path**: "data.currency.pp",
    - **Image**: "icons/commodities/currency/coin-inset-snail-silver.webp"
- **Name**: "Gold Coins",
    - **Path**: "data.currency.gp",
    - **Image**: "icons/commodities/currency/coin-embossed-crown-gold.webp"
- **Name**: "Electrum Coins",
    - **Path**: "data.currency.ep",
    - **Image**: "icons/commodities/currency/coin-inset-copper-axe.webp"
- **Name**: "Silver Coins",
    - **Path**: "data.currency.sp",
    - **Image**: "icons/commodities/currency/coin-engraved-moon-silver.webp"
- **Name**: "Copper Coins",
    - **Path**: "data.currency.cp",
    - **Image**: "icons/commodities/currency/coin-engraved-waves-copper.webp"

**Comment by GregorHak** [comment]
![obraz](https://user-images.githubusercontent.com/99994770/157747801-d54750a2-6ce2-4d8a-91da-d292010457f9.png)

**Comment by moo-man** [comment]
Sorry I wasn't aware of this discussion for some reason. Does the Item Piles support work?

**Comment by GregorHak** [comment]
Work, but background image look poorly (see image). Also, Container items do not work, you can't pack bottle to chest etc. Weapon, Ammunition, Armor and Trapping work well.
![obraz](https://user-images.githubusercontent.com/99994770/158051532-01297f69-7ebc-479d-ab8c-d9f8fbc84dfc.png)

**Comment by GregorHak** [comment]
I ask for improvments:
https://github.com/fantasycalendar/FoundryVTT-ItemPiles/issues/111

**Comment by GregorHak** [comment]
@moo-man can we get from You solution for styling of the window? It is easy thing to do? Or on your side it is impossible to do, or You have no time for this.
![obraz](https://user-images.githubusercontent.com/99994770/179741008-66aa7bc2-eb5b-4f15-b81b-ed8f366334b1.png)

**Comment by moo-man** [comment]
Yes generally module styling is done in the system
